app/routes.php

```diff
@@ -6,3 +6,5 @@
  * Date: 12/21/16
```

Comment:
The routes.php file will be deleted now. But its in the App directory, Should the folder be deleted as well?